src/cascadia/TerminalControl/ControlInteractivity.cpp

```diff
@@ -303,12 +306,12 @@ namespace winrt::Microsoft::Terminal::Control::implementation
                 // panning down)
                 const float numRows = -1.0f * (dy / fontSizeInDips.height<float>());
 
-                const auto currentOffset = ::base::ClampedNumeric<double>(_core->ScrollOffset());
-                const auto newValue = numRows + currentOffset;
+                const double currentOffset = ::base::ClampedNumeric<double>(_core->ScrollOffset());
+                const double newValue = numRows + currentOffset;

```

Comment:
I thought you'd want to keep `auto currentOffset`. That way you get a safe chromium math add for `newValue`.